spark-2.0-on-suse/azuredeploy.json

```diff
@@ -23,6 +23,57 @@
 	  "minValue": 1,
       "maxValue": 50
     },
+    "sparkSlavesSize": {
```

Comment:
let's leave this as a variable - it doesn't take anything away from the sample and makes template easier to maintain...